src/storage/s3.rs

```diff
@@ -168,9 +167,9 @@ pub(super) struct S3StorageTransaction<'a> {
 
 impl<'a> StorageTransaction for S3StorageTransaction<'a> {
     fn store_batch(&mut self, mut batch: Vec<Blob>) -> Result<(), Error> {
-        self.s3.runtime.handle().block_on(async {
+        self.s3.runtime.block_on(async {
             // Attempt to upload the batch 3 times
-            for _ in 0..3 {
+            for _ in 0usize..3 {
```

Comment:
Why did this have to change?

Reply:
I guess it didn't, I can't remember why I changed that.